src/server.ts

```diff
@@ -3,7 +3,9 @@ import express from 'express'
 import cors from 'cors'
 import helmet from 'helmet'
 import compress from 'compression'
+import errorHandler from '@middlewares/error-handler'
```

Comment:
Podríamos cambiar esto para mantener los nombres, ya que dice lo mismo que la función exportada pero alrevés 

```suggestion
import errorHandler from '@middlewares/error-handler'
import handleError from '@middlewares/error-handler'
```